server/templates/topic_page.html

```diff
@@ -31,7 +31,7 @@
 
 {% block content %}
 <div id="body" class="container-fluid"></div>
-<div id="metadata" class="d-none" data-place-dcid="{{ place_dcid }}" data-topic-id="{{ topic_id }}">
+<div id="metadata" class="d-none" data-place-dcid="{{ place_dcid }}" data-more-places="{{ more_places }}" data-topic-id="{{ topic_id }}">
```

Comment:
maybe we can just get the names of these places already and pass a list of NamedPlace objects

Reply:
That way, name fetch blocks the page rendering? 

I see the PageSelector had an UI (but commented out now) to pick places and fetch names. I wonder eventually these can be merged together.

Reply:
I think the rest of the apps that use subject pages blocks on getting names of places so blocking should be ok, but maybe this will also need some more thinking 

Yeah, I remember that, not sure if we're ever going to re-instate that UI though